warp10/src/main/java/io/warp10/continuum/sensision/SensisionConstants.java

```diff
@@ -316,6 +316,11 @@
   public static final String SENSISION_CLASS_CONTINUUM_STANDALONE_DELETE_DATAPOINTS = "warp.standalone.delete.datapoints";
 
   /**
+   * Number of datapoints deleted by 'standalone' delete per owner and application
+   */
+  public static final String SENSISION_CLASS_CONTINUUM_STANDALONE_DELETE_DATAPOINTS_PEROWNER = "warp.standalone.delete.datapoints.perowner";
```

Comment:
I've been having second thoughts, it's probably better to have a perowner AND a perapp set of classes rather than a combo app/owner, this would be more in sync with the rest of the exposed metrics.

Do you need the combo?